components/brave_ads/core/internal/account/user_data/README.md

```diff
@@ -28,6 +28,7 @@ Should only be included in the confirmation payload for users who have joined Br
 | segment  | yes  | no  | Advertising segment taxonomy for the chosen ad.  |
 | studies  | no  | no  | User studies. See [Griffin](https://github.com/brave/brave-browser/wiki/Brave-Variations-(Griffin)).  |
 | systemTimestamp  | no  | yes  | Privacy-preserving [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) timestamp.  |
+| topSegment  | yes  | no  | Top segment taxonomies.  |
```

Comment:
> "Top segment taxonomies."

Is it sent as text (e.g. "travel-air travel"), or is it some kind of non-human readable identifier?

Reply:
Sent as plaint text, i.e., from the list of segments I posted earlier. Thanks